src/test/javascript/spec/component/text-submission-assessment/text-submission-assessment.component.spec.ts

```diff
@@ -260,7 +269,10 @@ describe('TextSubmissionAssessmentComponent', () => {
         const alertService = TestBed.inject(AlertService);
         const errorStub = jest.spyOn(alertService, 'error');
 
+        component['textBlockRefs'] = [];
```

Comment:
What do you do here? That looks off. If you need to set a default value, maybe we need it in production code already. Otherwise, please document why this is required.

Reply:
I've done this a different way now. I don't really get why the test was passing before, but maybe it wasn't really correct before as well and some change with Jest now triggered that issue